plugins/modules/zos_mount.py

```diff
@@ -0,0 +1,995 @@
+                  be between 0 and 65535. Other than this, the value is not
+                  checked as being valid and the corresponding code page is not
+                  checked as being installed.
+        type: int
+        required: False
+    allow_uids:
+        description:
+            - >
+              Specifies whether the SETUID and SETGID mode bits on executables in
+              this file system are considered. Also determines whether the APF
+              extended attribute or the Program Control extended attribute is
+              honored.
+            - >
+              If I(True) the SETUID and SETGID mode bits are considered when a
+              program in this file system is run. SETUID is the default.
+            - >
+              If I(False) the SETUID and SETGID mode bits are ignored when a
+              program in this file system is run. The program runs as though the
+              SETUID and SETGID mode bits were not set. Also, if you specify the
+              NOSETUID option on MOUNT, the APF extended attribute and the Program Control
+              Bit values are ignored.
+        type: bool
+        required: False
+        default: True
+    sysname:
+        description:
+            - >
+              For systems participating in shared file system, SYSNAME specifies
+              the particular system on which a mount should be performed. This
+              system will then become the owner of the file system mounted. This
+              system must be IPLed with SYSPLEX(YES).
+            - >
+              IBM® recommends that you omit the SYSNAME parameter or specify
+              system_name where system_name is the name of this system.
+            - >
+              sysname is a 1–8 alphanumeric name of a system participating in shared file system.
+        type: str
+        required: False
+    automove:
+        description:
+            - >
+              These parameters apply only in a sysplex where systems are exploiting
+              the shared file system capability. They specify what is to happens to
+              the ownership of a file system when a shutdown, PFS termination, dead
+              system takeover, or file system move occurs. The default setting is
+              AUTOMOVE where the file system will be randomly moved to another system
+              (no system list used).
+            - >
+              I(AUTOMOVE) indicates that ownership of the file system can be
+              automatically moved to another system participating in a shared file system.
+            - >
+              I(NOAUTOMOVE) prevents movement of the file system's ownership in some situations.
+            - >
+              I(UNMOUNT) allows the file system to be unmounted in some situations.
+        type: str
+        choices:
+            - AUTOMOVE
+            - NOAUTOMOVE
+            - UNMOUNT
+        required: False
+        default: AUTOMOVE
+    automove_list:
```

Comment:
should this parameter be abstracted further?

Reply:
Not sure how to... it's an option that fires if a 'move' is called in the sysplex.